masonry/src/widget/widget_ref.rs

```diff
@@ -172,31 +172,32 @@ impl<'w> WidgetRef<'w, dyn Widget> {
     ///
     /// **pos** - the position in local coordinates (zero being the top-left of the
     /// inner widget).
-    pub fn find_widget_at_pos(&self, pos: Point) -> Option<WidgetRef<'w, dyn Widget>> {
+    pub fn find_widget_at_pos(&self, mut pos: Point) -> Option<WidgetRef<'w, dyn Widget>> {
```

Comment:
I think it would be better to shadow `pos` within the function explicitly than have this be `mut`

Reply:
This is non-blocking, but agreed. It's the pattern the rest of the codebase uses.

Reply:
Done